### PR TITLE
Organization management List Added  Task #196781 Task #196782

### PR DESCRIPTION
**Pull request by poojakarma**
Procedure :
Step: 1 create one component for Organization LIst
Step: 2 create one service to call API for  Organization list
Step: 3 Add one new tab for organization LIst
step: 4 Add table to show Organization list 
step: 5 Get organization data from API call 
step: 6 Show data in table with searching .sorting and paginating features

Implementation Video Link : 
https://drive.google.com/file/d/1LCclHWAjgtAraULkNapJoefxnRhvO4PU/view?usp=share_link

### CODE REVIEW

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                        <i class="pi pi-search"> </i>
+                        <input type="text" pInputText placeholder="Search"
+                            (input)="myTab.filterGlobal($any($event.target).value,'contains')" />
+                    </span>
+                </ng-template>
+                <ng-template pTemplate="header">
+                    <tr>
+                        <th scope="col" pSortableColumn="orgName">{{ 'ORGANIZATION_NAME' | i18next }}<p-sortIcon
+                                field="orgName"></p-sortIcon></th>
+                        <th scope="col" pSortableColumn="channel">{{ 'CHANNEL NAME' | i18next }}<p-sortIcon
```
Comment (gouravmore) [inline src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html]: Is this language constant working? with space in string.

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                                field="channel"></p-sortIcon></th>
+                        <th scope="col" pSortableColumn="id">{{ 'CHANNEL ID' |i18next }}<p-sortIcon
```
Comment (gouravmore) [inline src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html]: Is this language constant working? with space in string.

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  getAllOrganizationList() {
+    const body = {
+      "request": {
+        "filters": {
+          "isRootOrg": true
+        }
+      }
+
+    }
+    this.subscription = this.orgList.getAllOrganizationList(body).subscribe((data: any) => {
```
Comment (gouravmore) [inline src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts]: Need to add error block if any

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html (hunk 1 above)
Comment (poojakarma) [reply src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html]: yes sir this is working

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html (hunk 2 above)
Comment (poojakarma) [reply src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html]: yes sir working

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html (hunk 1 above)
Comment (gouravmore) [reply src/app/sb-admin/components/pages/sb-organization/sb-organization.component.html]: Add an underscore in between. Space should not be there in language constant.

**Review by gouravmore — APPROVED** [review]

**Pull request merged**